src/rdgnu-backend/.gitignore

```diff
@@ -0,0 +1,34 @@
+# compiled output
```

Comment:
Is there a reason this needs it's own .gitignore? I feel like we should be adding this to a project level .gitignore